packages/common/package.json

```diff
@@ -10,5 +10,8 @@
     "build": "tsup src/index.ts --format=esm --dts",
     "lint": "eslint --cache --ignore-pattern dist .",
     "typecheck": "tsc --noEmit"
+  },
+  "devDependencies": {
+    "mrmime": "^1.0.1"
```

Comment:
Since this is an external dependency, we can move it to deps (no need to bundle it)

Reply:
Since we don't publish `@lagon/common`, we have to bundle `mrtime` here to keep the cli running successfully.

Actually, I think it makes sense to publish `@lagon/common` like the other packages in our monorepo.

Reply:
Installing the cli will also install `mrtime`, similar to the other dependencies in `@lagon/cli` (chalk, commander, inquirer...). Those dependencies are not bundled.

But I agree that publishing `@lagon/common` might be a good solution instead of bundling it in the CLI. Let me think more about it

Reply:
Because the code of `@lagon/common` is bundle into `@lagon/cli`, the user will not install `@lagon/common` when using `@lagon/cli`, nor will the `mrtime` it depends on ~

Reply:
Oh ok, I understand what you wanted to say. I've tested to build `@lagon/cli` locally based on this PR but with `mrmime` moved from devdeps to deps. The built output contains `mrmime` bundled into it:

![Screenshot 2022-07-12 at 19 24 58](https://user-images.githubusercontent.com/43268759/178554707-ce8de2e9-f011-491e-b1e3-fdfc6bbb759f.png)

Reply:
Ummmm. This is the result of my local build, and I can see that mrtime is treated as external.
<img width="1227" alt="image" src="https://user-images.githubusercontent.com/41503212/178638366-185aaf2b-d4fd-41b5-8d9c-57eb6fb2aafc.png">

This is what the [documentation](https://tsup.egoist.sh/#excluding-packages) says, 
<img width="1028" alt="image" src="https://user-images.githubusercontent.com/41503212/178638421-9407db9b-7442-4ee3-bd95-5bccca08d27a.png">

so maybe your results are related to this issue https://github.com/egoist/tsup/issues/600 ?

Reply:
Yeah, I do have the same output file by building `@lagon/common`. But my screenshot above was the output of building `@lagon/cli`, sorry if this wasn't clear enough. Could you try to build `@lagon/cli` and see if you get the same output as me?

Reply:
Sorry, I see what you mean now, it does include the dependencies used by `@lagon/common` when building `@lagon/cli`.

I'll move it to dependencies